tests/subsys/bluetooth/mesh/sensor_subsys/CMakeLists.txt

```diff
@@ -1,32 +1,29 @@
 #
-# Copyright (c) 2021 Nordic Semiconductor ASA
+# Copyright (c) 2024 Nordic Semiconductor ASA
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 cmake_minimum_required(VERSION 3.20.0)
 
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
-project(bt_mesh_sensor_subsystem_test)
+project(bt_mesh_sensor_subsystem_new_test)
```

Comment:
Can probably drop the `_new` suffix now that we only have one set of tests again? Goes both here and in yaml and C files.

Reply:
removed *new*